testing/ceph/APKBUILD

```diff
@@ -0,0 +1,541 @@
+arch="all"
+license="LGPL-2.1 and CC-BY-SA-1.0 and GPL-2.0 and BSL-1.0 and GPL-2.0-with-autoconf-exception and BSD-3-Clause and MIT"
+depends="
+	ceph-osd
+	ceph-mds
+	ceph-mon
+	binutils
+"
+
+makedepends="
+	autoconf
+	automake
+	acl-dev
+	argp-standalone
+	binutils
+	boost-dev
+	btrfs-progs
+	build-base
+	bzip2-dev
+	ccache
+	cmake
+	coreutils
+	cryptsetup
+	curl-dev
+	cython
+	cython-dev
+	diffutils
+	eudev-dev
+	expat-dev
+	fcgi-dev
+	findutils
+	flex
+	fuse-dev
+	git
+	grep
+	jq
+	keyutils-dev
+	leveldb-dev
+	libaio-dev
+	libatomic_ops-dev
+	libedit-dev
+	libtirpc-dev
+	libtool
+	libxml2-dev
+	linux-headers
+	lvm2-dev
+	nss-dev
+	openldap-dev
+	openssl-dev
+	parted
+	procps
+	python-dev
+	py2-pip
+	py-nose
+	py-sphinx
+	py-virtualenv
+	readline-dev
+	rpcgen
+	sed
+	snappy-dev
+	userspace-rcu-dev
+	xfsprogs-dev
+	xmlstarlet
+	yasm
+"
+
+case $pkgrel in
+	''|*[!0-9]*)
+		_builddir="$srcdir"/$pkgname-$pkgver-`echo $pkgrel | sed 's/\./-/'`
+		_sourcepkg=ceph-$pkgver-`echo $pkgrel | sed 's/\./-/'`.tar.bz2
+		;;
+	*)
+		_builddir="$srcdir"/$pkgname-$pkgver
+		_sourcepkg="http://github.com/dx9/ceph/releases/download/$pkgver/$pkgname-$pkgver-beta1.tar.bz2"
+		;;
+esac
+
+source="
+	$_sourcepkg
+	$pkgname-common.post-deinstall
+	$pkgname-common.pre-install
+"
+
+subpackages="
+	$pkgname-base
+	$pkgname-common
+	$pkgname-mds
+	$pkgname-mon
+	$pkgname-fuse:ceph_fuse
+	$pkgname-radosgw
+	$pkgname-osd
+	$pkgname-doc
+	$pkgname-bash-completion:bash_completion
+	rbd-fuse:rbd_fuse
+	rbd-mirror:rbd_mirror
+	rbd-nbd:rbd_nbd
+	librbd
+	librbd-dev:librbd_dev
+	libcephfs
+	libcephfs-dev:libcephfs_dev
+	librados
+	librados-dev:librados_dev
+	librgw
+	librgw-dev:librgw_dev
+	py-rados:py_rados
+	libradosstriper
+	libradosstriper-dev:libradosstriper_dev
+	py-rbd:py_rbd
+	py-cephfs:py_cephfs
+"
+
+if [ "$WITH_TESTS" = ON ]; then
+	subpackages="$subpackages $pkgname-test:ceph_test"
+fi
+
+_prefix=/usr
+_bindir=$_prefix/bin
+_datadir=$_prefix/share
+_docdir=$_datadir/doc
+_includedir=$_prefix/include
+_libdir=$_prefix/lib
+_libexecdir=$_prefix/libexec
+_localstatedir=/var
+_mandir=$_datadir/man
+_sbindir=/usr/sbin
+_sysconfdir=/etc
+
+_udevrulesdir=/etc/udev/rules.d
+_python_sitelib=/usr/lib/python2.7/site-packages
+
+build() {
+   cd "$_builddir"
+   mkdir build
+   cd build
+   cmake .. \
+	   -DCMAKE_INSTALL_PREFIX=$_prefix \
+	   -DCMAKE_INSTALL_LIBDIR=$_libdir \
+	   -DCMAKE_INSTALL_LIBEXECDIR=$_libexecdir \
+	   -DCMAKE_INSTALL_LOCALSTATEDIR=$_localstatedir \
+	   -DCMAKE_INSTALL_SYSCONFDIR=$_sysconfdir \
+	   -DCMAKE_INSTALL_DOCDIR=$_docdir/ceph \
+	   -DCMAKE_INSTALL_MANDIR=$_mandir \
+	   -DWITH_REENTRANT_STRSIGNAL=ON \
+	   -DWITH_THREAD_SAFE_RES_QUERY=ON \
+	   -DWITH_MANPAGE=OFF \
+	   -DWITH_PYTHON3=OFF \
+	   -DWITH_TESTS=$WITH_TESTS
+   make
+}
+
+package() {
+	cd "$_builddir"
+	cd build
+	make DESTDIR="$pkgdir" install
+	cd ..
+
+	rm -f "$pkgdir"$_sysconfdir/init.d/ceph
+
+	install -D src/etc-rbdmap "$pkgdir"$_sysconfdir/ceph/rbdmap
```

Comment:
"install" installs as executable by default (ie 0755).
Please update this similar to the change in https://github.com/ceph/ceph/pull/11217